playbooks/common/openshift-cluster/upgrades/upgrade_scale_group.yml

```diff
@@ -50,11 +50,11 @@
     delegate_to: "{{ groups.oo_first_master.0 }}"
     register: l_upgrade_nodes_drain_result
     until: not (l_upgrade_nodes_drain_result is failed)
-    retries: "{{ 1 if openshift_upgrade_nodes_drain_timeout | default(0) == '0' else 0  | int }}"
+    retries: "{{ 1 if (openshift_upgrade_nodes_drain_timeout | default(0) | int == 0 ) else 0 }}"
```

Comment:
The ')' should go after 'int', not after the comparison.

Reply:
ok, updated